test/scrabble_test.rb

```diff
@@ -5,9 +5,31 @@
 require 'pry'
 
 class ScrabbleTest < Minitest::Test
+  attr_reader :game
+  def setup
+    @game = Scrabble.new
+  end
+
   def test_it_can_score_a_single_letter
-    skip
-    assert_equal 1, Scrabble.new.score("a")
+    assert_equal 1, game.score("a")
     assert_equal 4, Scrabble.new.score("f")
   end
+
+  def test_it_can_score_words
```

Comment:
Would like to see a test for `nil` and a test for `""`. Those were earlier in the description, and currently you're not solving for those cases (believe empty string will work, but nil will error out on line 33).